app/api/process.py

```diff
@@ -56,11 +56,7 @@
     'MD': [utils.standardize_data, utils.post_processing],
     'ME': [utils.standardize_data, aggregate_outbreaks.collapse_outbreak_rows, utils.post_processing],
     'MI': [utils.standardize_data, utils.post_processing],
-    'MN': [
-        utils.standardize_data,
-        lambda df: aggregate_outbreaks.collapse_facility_rows_no_adding(
-            df, use_facility_type_to_group=False),
-        utils.post_processing],
+    'MN': [utils.standardize_data, drop_no_data.drop_no_data, utils.post_processing],
```

Comment:
excellent, thank you! Also I think I remember you saying we might want to do this for FL too - should we add that here if so?

Reply:
thanks for the reminder! I'm already doing this with a FL-specific function in the aggregate_outbreaks file but I'll replace that with this, it'll be cleaner!

Reply:
I'm going to hold on this until I hear back from the LTC team on some questions. At present my FL-specific function includes outbreak status in the columns to check and I'm looking for some clarity about whether we wish to keep that context or not.